posts/2021-11-26-jakarta-ee-9.1.adoc

```diff
@@ -13,19 +13,16 @@ open-graph-image: https://openliberty.io/img/twitter_card.jpg
 = Jakarta EE 9.1 support and configurable response headers in Open Liberty 21.0.0.12
 Ryan Storey <https://github.com/ryan-storey>
 :imagesdir: /
-:url-prefixx
+:url-prefix:
 :url-about: /
 //Blank line here is necessary before starting the body of the post.
 
 Open Liberty 21.0.0.12 offers exciting new enhancements such as Jakarta EE 9.1 support, and configurable response headers, which offer more granular control over response headers. This release also provides several significant bug fixes.
-
+ 
 In link:{url-about}[Open Liberty] 21.0.0.12:
 
 * <<jee_9.1, Jakarta EE 9.1 support>>
 * <<response_headers, Configurable Response Headers>>
-** <<SUB_TAG_1, SUB_FEATURE_1_HEADING>>
-** <<SUB_TAG_2, SUB_FEATURE_2_HEADING>>
-* <<TAG_3, FEATURE_3_HEADING>>
 
 View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A21.0.0.12+label%3A%22release+bug%22[21.0.0.12].
```

Comment:
link is wrong. this seems to happen frequently for the release blog posts.... can you update the template so that the link doesn't use RELEASE_VERSION and makes it clear that the version is of the X00X format (no periods)